src-built-in/components/toolbar/stores/searchStore.js

```diff
@@ -49,9 +50,11 @@ var Actions = {
 	},
 	setFocus(bool, target) {
 		focus = bool;
+		if (!menuWindow) return;
```

Comment:
I'm of the opinion that you should always do 

```
if (condition) {
}
```

Not a big deal if it's already done in the codebase and we don't have rules against it.